<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2330</width>
    <height>2395</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>1301</width>
      <height>761</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="functionalityGrid">
     <item row="0" column="1">
      <widget class="QLabel" name="gbSelectionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="text">
        <string>Gameboard selection</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="playArea" native="true">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>781</width>
          <height>621</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="hLayout" rowstretch="0" columnstretch="0">
         <property name="sizeConstraint">
          <enum>QLayout::SetFixedSize</enum>
         </property>
         <property name="horizontalSpacing">
          <number>6</number>
         </property>
         <property name="verticalSpacing">
          <number>1</number>
         </property>
        </layout>
       </widget>
       <widget class="QListWidget" name="boardsList">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>10</y>
          <width>371</width>
          <height>600</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>600</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777214</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
       <widget class="QListWidget" name="playerList">
        <property name="geometry">
         <rect>
          <x>1160</x>
          <y>10</y>
          <width>1493</width>
          <height>769</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>600</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="scoreListLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
       <property name="text">
        <string>Scorelist</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>674</width>
      <height>243</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item row="1" column="2">
      <widget class="QLabel" name="timerLabel">
       <property name="text">
        <string>Timer (min : s)</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLineEdit" name="cardEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="timerLabel_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="quitButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLineEdit" name="playerEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="resetButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="startButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="enterButton">
       <property name="text">
        <string>Enter</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="turnButton">
       <property name="text">
        <string>End Turn</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="errorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="playerLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="enterLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLCDNumber" name="lcdNumberMin">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Noto Serif Tibetan</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>2</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLCDNumber" name="lcdNumberSec">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>100</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Noto Serif Tibetan</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>2</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Flat</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
